packages/charts/examples/components/demo/chart-types/spark/chart-docs-spark.component.html

```diff
@@ -115,6 +115,19 @@ <h3>update vs. updateSparks</h3>
     <nui-spark-chart-multiple-example></nui-spark-chart-multiple-example>
 </nui-example-wrapper>
 
+<h2>Area with Multiple Series</h2>
```

Comment:
I think we can leave out the heading here since we're still inside the overall Multiple Series heading, and the example wrapper has the label for what it's showing. What do you think?
```suggestion
```

Reply:
Sounds good to me!